coder_sniffer/Drupal/Sniffs/Semantics/RemoteAddressSniff.php

```diff
@@ -42,9 +42,10 @@ public function register()
      */
     public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
     {
-        $string = $phpcsFile->getTokensAsString($stackPtr, 4);
-        if ($string === '$_SERVER["REMOTE_ADDR"]' || $string === '$_SERVER[\'REMOTE_ADDR\']') {
-            $error = 'Use the function ip_address() instead of $_SERVER[\'REMOTE_ADDR\']';
+        $string           = $phpcsFile->getTokensAsString($stackPtr, 4);
+        $startOfStatement = $phpcsFile->findStartOfStatement($stackPtr);
+        if (($string === '$_SERVER["REMOTE_ADDR"]' || $string === '$_SERVER[\'REMOTE_ADDR\']') && $stackPtr !== $startOfStatement) {
+            $error = 'Use Drupal::request()->getClientIp() instead of $_SERVER[\'REMOTE_ADDR\']';
```

Comment:
same here, should be "Use Drupal::request()->getClientIp() or ip_address() (Drupal 7) instead of $_SERVER[\'REMOTE_ADDR\']"

Reply:
Done!